darwin/importer/importer.py

```diff
@@ -1308,6 +1309,10 @@ def _import_annotations(
 
         if annotation.id:
             serial_obj["id"] = annotation.id
+        else:
+            annotation_id = str(uuid.uuid4())
+            annotation.id = annotation_id
+            serial_obj["id"] = annotation_id
```

Comment:
This gives every annotation an ID if one isn't present, even those that we're not importing properties to. Because of this, I'm not extremely happy with it as a solution, but it is by far the simplest solution I could think of. IDs have to be added here before `_import_properties()` & `_update_payload_with_properties()`

I don't think inefficiency this will have a significant impact. However, I'm open to feedback on alternative approaches!